examples/go/google-cloudrun/README.md

```diff
@@ -0,0 +1,3 @@
+# google-cloudrun
+
+A port of the TypeScript example at: https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/google-cloudrun
```

Comment:
I think it would be fine to just copy & paste the README from the other example here.
Saves readers a click 😁

Reply:
Thanks, I've done that. The README wasn't particularly relevant, so I've updated it too.